contracts/AllDay.cdc

```diff
@@ -82,6 +82,17 @@ access(all) contract AllDay: NonFungibleToken {
     access(all) event MomentNFTMinted(id: UInt64, editionID: UInt64, serialNumber: UInt64)
     access(all) event MomentNFTBurned(id: UInt64)
 
+    // Badges Events
+    //
+    access(all) event BadgeCreated(slug: String, title: String, description: String, visible: Bool, slugV2: String, metadata: {String: String})
+    access(all) event BadgeUpdated(slug: String, title: String, description: String, visible: Bool, slugV2: String, metadata: {String: String})
+    access(all) event BadgeAddedToPlay(badgeSlug: String, playID: UInt64, metadata: {String: String})
```

Comment:
Nit: use more specific name for metadata param to avoid confusion with badge metadata field, same for BadgeAddedToEdition and BadgeAddedToMoment

Reply:
hmm, i think it is fine. The events mirror the tables we have in nfl and eventually atlas so it's clear that handling any one event should only change data meant for that table